debezium-core/src/main/java/io/debezium/transforms/ByLogicalTableRouter.java

```diff
@@ -24,20 +22,22 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
-import io.debezium.config.Configuration;
-import io.debezium.config.Field;
-import io.debezium.data.Envelope;
-import io.debezium.util.AvroValidator;
+import java.util.Map;
+import java.util.Objects;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+import static org.apache.kafka.connect.transforms.util.Requirements.requireStruct;
```

Comment:
I don't think it's a good idea to reorder the imports. They used to match the order used in other classes (e.g., statics, JDK, other libs, Debezium's), so with these changes the next person to modify the class will likely reorder again if their IDE preferences are different. Generally it's better to try to match the preferences already used in the project.